src/odemis/driver/symphotime.py

```diff
@@ -887,3 +920,37 @@ def _setDirectory(self, value):
             raise ValueError("Directory name too long. Cannot be longer than 63 characters")
         return value
 
+
+class DetectorLive(model.Detector):
+    '''
+    Detector that is a child of the Symphotime parent. Provides a stream of count values.
+
+    Parameters:
+
+        data: BasicDataFlow that can be subscibred to
+
+    '''
+
+    def __init__(self, name, role, parent, **kwargs):
+        '''
+        parent (symphotime.Controller): a symphotime server parent object
+        '''
+        super(DetectorLive, self).__init__(name, role, **kwargs)
+
+        if isinstance(parent, Controller):
```

Comment:
No need to check. Keep the code simple, and just expect that anyone ever uses that code, it has read the one line documentation.